src/android/post_processor_jpeg.cpp

```diff
@@ -0,0 +1,263 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
```

Comment:
Please drop this file is a copy of `src/android/jpeg/post_processor_jpeg.cpp` and as far as I can tell doesn't get compiled in

Reply:
No it does, this is eventually where the process crash